python/cli/prelude_cli/views/build.py

```diff
@@ -41,6 +41,7 @@ def create_template(template, name):
         template_body = template_body.replace('$ID', t['id'])
         template_body = template_body.replace('$NAME', t['name'])
         template_body = template_body.replace('$UNIT', t['unit'])
+        template_body = template_body.replace('$TECHNIQUE', t['technique'])
```

Comment:
think this'll error if technique is none. might need to `str(t['technique'])` or smth